systemtests/src/main/java/io/enmasse/systemtest/KeycloakClient.java

```diff
@@ -259,6 +263,7 @@ public void deleteUser(String realm, String userName) throws Exception {
                 throw new RuntimeException(String.format("User {} is not deleted from keycloak", userName));
             }
         }
+        TimeMeasuringSystem.stopOperation(operationID);
```

Comment:
What happens if the exception is thrown and the stopOperation is never called? Just bad data? Consider wrapping it in a try{ } finally

Reply:
yeah just bad data 👍 good point I will update it